frontend_tests/puppeteer_tests/edit.ts

```diff
@@ -5,7 +5,7 @@ import common from "../puppeteer_lib/common";
 async function trigger_edit_last_message(page: Page): Promise<void> {
     await page.evaluate(() => {
```

Comment:
I am not sure why we are using `page.evaluate` to click on the option. Should I also fix it?